DumpCementeryDB.sql

```diff
@@ -16,8 +16,111 @@
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
 /*
-DB Structure Goes Here...
+DB Structure Goes Here...dwa dw w addwa  ftghger
 */
+
+/* ####### Create Sector ####### */
+
+DROP TABLE IF EXISTS `sektor`;
+CREATE TABLE `sektor` (
+  `id` int(11) NOT NULL AUTO_INCREMENT
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=31 DEFAULT CHARSET=utf8mb4;
```

Comment:
Auto increment should be 1 not 31 (same for lines) 43, 55, 70, 82, 95

Reply:
Ooooh, i havent noticed that. Omw